packages/@azure/servicebus/data-plane/package.json

```diff
@@ -23,6 +23,8 @@
   },
   "dependencies": {
     "@azure/amqp-common": "^1.0.0-preview.1",
+    "@azure/arm-servicebus": "^0.1.0",
+    "@azure/ms-rest-nodeauth": "^0.9.1",
```

Comment:
Since we are using these for just running the tests, we should move them under `devDependencies`. All packages under `dependencies` get shipped as part of the product to the user. We wouldn't want these to be downloaded to the user's machine when the `npm install` our package